github/security_advisories.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+type SecurityAdvisoriesService service
+
+// RequestCVE request a CVE for a repository security advisory.
```

Comment:
```suggestion
// RequestCVE requests a Common Vulnerabilities and Exposures (CVE) for a repository security advisory.
// The ghsaID is the GitHub Security Advisory identifier of the advisory.
```

Reply:
Done

Reply:
Sorry, this was not completed.  Please add the description shown above for "ghsaID".

Reply:
Sorry, for the back and forth. Completely missed this